build-logic/dependency-modules/src/main/kotlin/gradlebuild.dependency-modules.gradle.kts

```diff
@@ -70,6 +70,8 @@ dependencies {
             "org.gradle.profiler:gradle-profiler",
             setOf("gradle-tooling-api")
         )
+
+        all<JschReplacementVersionRule>()
```

Comment:
The change needed here is closer to what `ReplaceCglibNodepWithCglibRule` does and should be applied to `jgit` only.